backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -281,6 +282,39 @@ func (s *WorkService) GetCommitCount() fiber.Handler {
 	}
 }
 
+func (s *WorkService) GetCommitsPerDay() fiber.Handler {
+    return func(c *fiber.Ctx) error {
+        work, err := s.getWork(c)
+		if err != nil {
+			return err
+		}
+
+
+        
+        // Unsure how this will work for group assignments
+        var opts github.CommitsListOptions
+        opts.Author = work.Contributors[0].GithubUsername
+        commits, err := s.appClient.ListCommits(c.Context(), work.OrgName, work.RepoName, &opts)
+        if err != nil {
+            return errs.GithubAPIError(err)
+        }
+
+        commitDatesMap := make(map[time.Time]int)
+        for _, commit := range commits {
+            commitDate := commit.GetCommit().GetCommitter().Date
+
+            if (commitDate != nil) {
+                truncatedDate := time.Date(commitDate.Year(), commitDate.Month(), commitDate.Day(), 0, 0, 0, 0, commitDate.Location())
```

Comment:
Is this in UTC? I think standardizing our dates to always be UTC going in and out will make things easier

Reply:
Yah its UTC 00:00:00